### PR TITLE
Dashboard

### LINKED ISSUE #13: Dashboard
As an authenticated user,
When I visit '/dashboard'
I should see: 
- [x] 'Welcome `<username>`!' at the top of page
- [x] #9  A button to Discover Movies
- [x] #10 A friends section

### PR DESCRIPTION
**Pull request by rcasias**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Added a dashboard which includes a skeleton to the structure to the page as well as functionality to friends section.

* **What is the current behavior?** (You can also link to an open issue here)
page does not exist

* **What is the new behavior (if this is a feature change)?**
page created, structured and has ability to search for other users and has the ability to create friendships

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
no breaking changes

### All Submissions:

* [x] Have you followed the guidelines in our Contributing document?

### New Feature Submissions:

1. [x] Does your submission pass tests?
2. [x] Have you lint your code locally prior to submission?

### Changes to Core Features:

* [x] Have you written new tests for your core changes, as applicable?
* [x] Have you successfully ran tests with your changes locally?

- [x] All Tests are Passing

closes #13

### CODE REVIEW

File: spec/features/users/dashboards/show_spec.rb [hunk 1]
```diff
@@ -1,5 +1,58 @@
...
+  before(:each) do
+    user_1 = User.create!(email: "user1@email.com", password_digest: "1234")
+    visit '/'
+    click_on("New to Viewing Party? Register Here")
+    within 'form' do
+      fill_in 'user[email]', with: 'test2@test.com'
+      fill_in 'user[password]', with: 'password'
+      click_button 'Register'
+    end
+  end
```
Comment (redferret) [inline spec/features/users/dashboards/show_spec.rb]: You'll need to refactor this to utilize testing helpers for logging in and use FactoryBot to create your users

File: spec/models/user_spec.rb [hunk 2]
```diff
@@ -5,4 +5,28 @@
...
+
+  describe "relationships" do
+    it {should have_many :friendships}
+    it {should have_many(:friends).through(:friendships)}
+  end
+
+  it 'cand add a friend to a user' do
+    user_1 = User.create!(email: "user1@email.com", password_digest: "1234")
+    user_2 = User.create!(email: "user2@email.com", password_digest: "1234")
+    user_3 = User.create!(email: "user3@email.com", password_digest: "1234")
```
Comment (redferret) [inline spec/models/user_spec.rb]: Utilize FactoryBot

**Review by redferret — CHANGES_REQUESTED** [review]

File: spec/features/users/dashboards/show_spec.rb [hunk 3]
```diff
@@ -1,5 +1,58 @@
...
-end
\ No newline at end of file
+  before(:each) do
+    user_1 = User.create!(email: "user1@email.com", password_digest: "1234")
+    visit '/'
+    click_on("New to Viewing Party? Register Here")
+    within 'form' do
+      fill_in 'user[email]', with: 'test2@test.com'
+      fill_in 'user[password]', with: 'password'
+      click_button 'Register'
```
Comment (redferret) [inline spec/features/users/dashboards/show_spec.rb]: Utilize FactoryBot to create your 2 users and use `login_as(user)` helper for logging in

**Review by redferret — CHANGES_REQUESTED** [review]

File: spec/features/users/dashboards/show_spec.rb (hunk 1 above)
Comment (rcasias) [reply spec/features/users/dashboards/show_spec.rb]: thank you for walking me through that!

**Review by redferret — APPROVED** [review]
Great job!

**Pull request merged**